src/typed-method-types/workflows/triggers/tests/event_test.ts

```diff
@@ -85,13 +85,10 @@ Deno.test("Mock call for event", async (t) => {
             },
           });
           assertEquals(res.ok, true);
-          if (res.ok) {
-            assertEquals(res.trigger, event_response.trigger);
```

Comment:
I found this assertion to not be very useful. Since we control the contents of the response via the mocking utility earlier in the test, it will always be true. As a result, I removed it.